src/ray/core_worker/common.h

```diff
@@ -0,0 +1,35 @@
+#ifndef RAY_CORE_WORKER_COMMON_H
+#define RAY_CORE_WORKER_COMMON_H
+
+#include <string>
+
+#include "ray/id.h"
+#include "ray/common/buffer.h"
+
+namespace ray {
+
+/// Type of this worker.
+enum class WorkerType { WORKER, DRIVER };
```

Comment:
What other types make sense here, e.g., does it make sense to have an `ACTOR` type or a `LOCAL_MODE_DRIVER` type?

Reply:
Yes, these 2 make sense. That's why WorkerType is an enum, instead of a boolean `is_driver`. But I'd like to defer adding them when we actually implement the functionalities.